cpp/src/copying/slice.cpp

```diff
@@ -72,9 +71,7 @@ std::vector<cudf::table_view> slice(cudf::table_view const& input,
 {
   CUDF_FUNC_RANGE();
   CUDF_EXPECTS(indices.size() % 2 == 0, "indices size must be even");
-  std::vector<cudf::table_view> result{};
-
-  if (indices.size() == 0 or input.num_columns() == 0) { return result; }
```

Comment:
I assume that you dropped the `input.num_columns() == 0` intentionally and checking `empty()` on `indices` is equivalent?

Reply:
no, both were two different conditions. I added this condition while creating this API (Not sure why, I can't track back to any request and design), and it is normal to expect number of columns as number of slices requested. So, removed this change.

Reply:
I don't think the number of slices should match the number of columns. I think the number of slices is the number of output tables -- all output tables will have a slice across all columns.

Reply:
I meant in case of column is sliced, in case of table as you mentioned set of tables.